getting-started/templates/systemlink-values.yaml

```diff
@@ -357,6 +411,23 @@ testmonitorservice:
     #     ## NOTE: Ignored unless `database.connectionInfo.secretName` parameter is set.
     #     ##
     #     migrationPasswordKey: null
+
+    ## Service account for Test Monitor.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+    ##
+    serviceAccount:
+      ## @param serviceAccount.create Specifies whether a service account should be created
+      ##
+      create: false
```

Comment:
I think just exposing the annotations section in the top-level helm values should be enough. Do we have any other reason to expose other parameters?

Reply:
Removed 'serviceAccount.create' as it is true by default. The sericeAccount.name parameter would be useful while referencing the service account for authentication using workflow identity